### ISSUE TITLE
NoMethodError on Windows platforms when invoking Logging::Logger.trace

### ISSUE THREAD

**Issue by sanfrancrisko** [puppetlabs/bolt#2188]
## Describe the Bug
When running acceptance tests for a module on a Windows platform via Litmus, using `v2.28.0` of the bolt gem, we are seeing the following `NoMethodError` thrown:
```
     Failure/Error: LitmusHelper.instance.apply_manifest(pp)
     NoMethodError:
       undefined method `trace' for #<Logging::Logger:0x00007f9faa9df868>
       
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/analytics.rb:228:in `event'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:208:in `report_transport'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:121:in `block in queue_execute'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:119:in `each'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:119:in `flat_map'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:119:in `queue_execute'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:174:in `batch_execute'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:317:in `block in upload_file'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:182:in `log_action'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt/executor.rb:314:in `upload_file'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt_spec/run.rb:225:in `upload_file'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt_spec/run.rb:110:in `block in upload_file'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt_spec/run.rb:152:in `block in with_runner'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt_spec/run.rb:150:in `with_runner'
     # ./.bundle/gems/ruby/2.5.0/gems/bolt-2.28.0/lib/bolt_spec/run.rb:109:in `upload_file'
     # ./.bundle/gems/ruby/2.5.0/gems/puppet_litmus-0.18.4/lib/puppet_litmus/puppet_helpers.rb:139:in `block in create_manifest_file'
     # ./.bundle/gems/ruby/2.5.0/gems/honeycomb-beeline-2.2.0/lib/honeycomb/client.rb:68:in `start_span'
     # ./.bundle/gems/ruby/2.5.0/gems/puppet_litmus-0.18.4/lib/puppet_litmus/puppet_helpers.rb:119:in `create_manifest_file'
     # ./.bundle/gems/ruby/2.5.0/gems/puppet_litmus-0.18.4/lib/puppet_litmus/puppet_helpers.rb:71:in `block in apply_manifest'
     # ./.bundle/gems/ruby/2.5.0/gems/honeycomb-beeline-2.2.0/lib/honeycomb/client.rb:68:in `start_span'
     # ./.bundle/gems/ruby/2.5.0/gems/puppet_litmus-0.18.4/lib/puppet_litmus/puppet_helpers.rb:41:in `apply_manifest'
     # ./spec/spec_helper_acceptance_local.rb:29:in `setup_test_directory'
     # ./spec/acceptance/backup_spec.rb:5:in `block (2 levels) in <top (required)>'
```

## Expected Behaviour
There should be `NoMethodError` exception thrown from Bolt.

## Steps to Reproduce
Steps to reproduce the behaviour:

- Clone and check out the [`puppetlabs-iis`](https://github.com/puppetlabs/puppetlabs-iis) module (_PLEASE NOTE: `puppetlabs-concat` was the original module the issue was discovered in by @daianamezdrea - any module running acceptance tests on Windows should be OK_)
- `bundle install`
- `bundle exec rake 'litmus:provision_list[default]'` (_PLEASE NOTE: You'll need Vagrant and either VirtualBox/VMWare Fusion installed for this step to work_)
- `bundle exec rake 'litmus:install_agent'`
- `bundle exec rake 'litmus:install_module'`
- `bundle exec rake 'litmus:acceptance:parallel'`

## Environment
 - **Version:** `2.28.0`
 - **Platform** All Windows OSs

## Additional Context
- Unix OSs seem to be unaffected
- Looks very similar to this issue: https://github.com/puppetlabs/bolt/issues/2117

**Comment by tphoney** [comment]
I am seeing a similar issue @sanfrancrisko

**Comment by lucywyman** [comment]
It sounds like this was fixed by #2191 - can we close this issue?

**Comment by sanfrancrisko** [comment]
@lucywyman We can - I thought [my comment here using the 'Closes' keyword](https://github.com/puppetlabs/bolt/pull/2191#issuecomment-694910471) would have done that automatically when that PR was merged, but apparently not.
